frontend/app/src/components/display/AdaptiveWrapper.vue

```diff
@@ -9,33 +9,32 @@
     <slot />
   </component>
 </template>
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script setup lang="ts">
+import { get } from '@vueuse/core';
+import { computed, toRefs } from 'vue';
```

Comment:
just FYI, after yesterday's PR this should be in `auto-import.d.ts` so we should be able to skip the imports, no need to change anything right now just for future reference.